src/Fixer/AbstractPhpUnitFixer.php

```diff
@@ -153,13 +153,20 @@ private static function isPreventedByAttribute(Tokens $tokens, int $index, array
             return false;
         }
 
-        $attributeIndex = $tokens->getPrevMeaningfulToken($index);
-        if (!$tokens[$attributeIndex]->isGivenKind(CT::T_ATTRIBUTE_CLOSE)) {
+        $modifiers = [T_FINAL];
```

Comment:
potentially we can reuse bit of `getDocBlockIndex` , where we jump directly before any final/readonly/...` to expected docblock place.